osu.Server.Spectator/Hubs/MultiplayerHub.cs

```diff
@@ -140,6 +140,8 @@ public async Task<MultiplayerRoom> JoinRoomWithPassword(long roomId, string pass
                     }
                 }
 
+                await room.CountdownImplementation.UpdateTimeRemaining();
```

Comment:
On a similar note to my above comment, I don't really like this call calling an `Update` methods on a property of the room, feels too deep?

Reply:
Addressed in comment above.